scripts/wdl/README.md

```diff
@@ -6,5 +6,70 @@ https://github.com/broadinstitute/wdl
+
+*For segmenter parameters, see the explanation in the Segmenter tool or page 11 of https://bioconductor.org/packages/release/bioc/manuals/DNAcopy/man/DNAcopy.pdf*
+- ``case_gatk_acnv_workflow.seg_param_alpha`` -- (Advanced)  Segmenter parameter, default=0.01 
+- ``case_gatk_acnv_workflow.seg_param_nperm`` -- (Advanced)  Segmenter parameter, default=10000 
+- ``case_gatk_acnv_workflow.seg_param_pmethod`` -- (Advanced)  Segmenter parameter, default=HYBRID 
+- ``case_gatk_acnv_workflow.seg_param_minWidth`` -- (Advanced)  Segmenter parameter, default=2 
+- ``case_gatk_acnv_workflow.seg_param_kmax`` -- (Advanced)  Segmenter parameter, default=25 
+- ``case_gatk_acnv_workflow.seg_param_nmin`` -- (Advanced)  Segmenter parameter, default=200 
+- ``case_gatk_acnv_workflow.seg_param_eta`` -- (Advanced)  Segmenter parameter, default=0.05 
+- ``case_gatk_acnv_workflow.seg_param_trim`` -- (Advanced)  Segmenter parameter, default=0.025 
+- ``case_gatk_acnv_workflow.seg_param_undoSplits`` -- Segmenter parameter, default=NONE   **For WGS: SDUNDO**
+- ``case_gatk_acnv_workflow.seg_param_undoPrune`` -- (Advanced)  Segmenter parameter, default=0.05 
+- ``case_gatk_acnv_workflow.seg_param_undoSD`` -- Segmenter parameter, default=3 **For WGS: 2**
+
+
+#### Explanation of fields in PoN workflow
+
+Fields marked as Advanced almost never require a user to adjust the default value.
+Reference used must be the same between PoN and case samples.
+
+- ``pon_gatk_workflow.disable_sequence_dictionary_validation`` -- (Advanced) "true",
+- ``pon_gatk_workflow.isWGS`` -- Change to "true"" if running WGS samples
+- ``pon_gatk_workflow.max_open_files`` -- (Advanced) "100",  Maximum number of files to combine simultaneously while combining read counts 
+- ``pon_gatk_workflow.wgsBinSize`` --  Size of bins (in bp) for WGS read counts CLI.  *This must be the same values used for all case samples*.  Ignored if not running WGS.
+- ``pon_gatk_workflow.enable_gc_correction`` --  "true" is recommended,  true/false option change to false to bypass gc bias correction step.  All case samples must be run with the same value as used in PoN creation.
+- ``pon_gatk_workflow.ref_fasta`` -- Path to reference fasta file. Broad internal: /seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta
+- ``pon_gatk_workflow.ref_fasta_fai`` --  Path to reference fasta fai file.  Broad internal: /seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta.fai
+- ``pon_gatk_workflow.pon_entity_id`` --  Name of the final PoN file
+- ``pon_gatk_workflow.normal_bam_list`` --  tsv file consisting of corresponding bam and corresponding index files as described in pon_gatk_workflow.wdl
+- ``pon_gatk_workflow.ref_fasta_dict`` --  Path to reference dict file.  Broad internal: /seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.dict
+- ``pon_gatk_workflow.target_file`` --  Target file (NOT in bed format) that was used to describe the baits in capture (exome) samples.
```

Comment:
I would say something here about using `ConvertBedToTargetFile` if their target file is a bed file.